nautobot/core/templates/generic/object_detail.html

```diff
@@ -13,9 +13,11 @@
         <div class="col-sm-8 col-md-9">
             <ol class="breadcrumb">
             {% block breadcrumbs %}
+                {% if list_url %}
```

Comment:
What's the story on these changes? Is this needed because we don't have a list view for CircuitTerminations at this time?

Reply:
That is correct, I didn't think it made sense to me to add a list view. This also allows plugins or other objects that don't need a list view to use the generic object detail.